### PR TITLE
pypi and setuptools

### PR DESCRIPTION
**Pull request by jonathanrcross**
- Update encrypted token to deploy to pypi change condition to python 3.7 python.
- Switch from disutils to setuptools in setup.py

Unsure if travis is using twine for pypi upload, possibly why it failed on python 2.7?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+    secure: Wwy6E8My1CXssswhsVOZZDnEL/cGYFnZVF/Pq/X9T1DmRuVnCDOMmZ43t10ASXt4C7pCUVdhmlmnwbyOvNHbhMQ79EXH7Bqno7AnP2C8mC+l7u3KXqrXpvCQ+lIno8JwS3Wp3DQas+ROQ3VlyLenqoR+4bihWDQc573wwH8XM+9RM9jJcuwg2xc30sJAxG624loyD6S4BjMIit2ND/+MgupbiG69nBx9yWLrdJKj1BF0YO9hmumZ928asM+UdtrU2QtVT84g8I2aC5tM34kRZRiM/OQdApm93BVGwlo1hmAQ1YHJevZflMGSAouJjieOVxl4hkN+gYVyHxO1YjnzjgCTWzbKMtmCr7KtputclRwrf7wrsxl6FRn2HmOKPxLfY9CnVjv6r2e64iiYWmj4g8J/OgI1TD8B6k4e/ca2Oen/iu8/uiZWLT0XEOAtWLxpAc7kGwcfbXnfVNRUPgQY0ePLsI+Lp3VY7hh65YQQDaHqH+/Ib/MNS1XAhZMrMM1iJ4YgNMsmLuBXXI63Ef3T/T4Tw7plydw6jFy2Z2tyIG8/IFnV4t8CofvThX/Wfof8fX0YphuIFPGJZVzVRW+FSQbx8iniTDeu1Q7d76RxXYdoMd5Aesbn/dwWX22KLHO/x2gKlBNjPIVuvx4XxIsQGsan/7u1582MEC681AjggE4=
   on:
     tags: true
     repo: jrxFive/python-nomad
-    condition: $TRAVIS_PYTHON_VERSION = "2.7"
-
+    condition: $TRAVIS_PYTHON_VERSION = "3.7"
```
Comment (jeffwecan) [inline .travis.yml]: Hmm yeah, the one working example I have of a deploy stage in Travis uses python 3.7 so maybe that's it 🤷‍♂️

**Comment by jeffwecan** [comment]
Probably need to retrigger those builds to get the correct status reflected here (if you care to). travis.org was having a bit of an outage yesterday along those lines...

**Comment by jeffwecan** [comment]
Oh! And for what it's worth, when I needed to validate travis + pypi publishing logic in the past, I have used a corresponding entry on https://test.pypi.org/ / https://packaging.python.org/guides/using-testpypi/. In my situation, I added a commit in another branch with a commit to point Travis at the test server url, pushed that up to a fork, and then did a tagged release to trigger the deploy condition. 

Ref: https://docs.travis-ci.com/user/deployment/pypi/#releasing-to-a-self-hosted-pypi

**Comment by jonathanrcross** [comment]
Yea def need to check out the test pypi! If this doesn't work on merge will do that with a follow up branch...

**Pull request merged**